doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,19 +68,86 @@ between tenants
 - Enable Role-Based Access Control (RBAC)
 
 ## 5.3 Node Hardening
-Ensure Kubernetes nodes are secure, hardened and configured correctly following
-well known security framework e.g. CIS benchmark, etc. Restrict administrative
-access to Kubernetes nodes while avoiding direct access to nodes for operational
-activities including debugging, troubleshooting, and other tasks.
-
-## 5.4 Authentication & Authorisation
-Secure all connections to a Kubernetes Cluster. Adopt the following security
-authentication mechanisms:
- - Configure user roles and access levels to provide segregation of duties (RBAC)
- - Use multi-factor authentication for all administrative access
- - Use token-based or certificate-based service and session authentication mechanisms
- - Integrated with existing identity management platforms e.g SAML, AD, etc. for
- access control
+### 5.3.1 Node hardening: Securing Kubernetes hosts 		
+Security benchmarking
+When an operating system or application is installed, it comes with default settings. Usually, all ports are open, and all application services are turned on. In other words, freshly installed assets are highly insecure.
+
+Ensure Kubernetes nodes are secure, hardened and configured correctly following well known security framework. Security benchmarks, for example CIS benchmarks are a set of configuration standards and best practices designed to help ‘harden’ the security of their digital assets. 
+
+### 5.3.2 Restrict direct access to nodes
+Restrict root/administrative access to Kubernetes nodes while avoiding direct access to nodes for operational activities including debugging, troubleshooting, and other tasks.
```

Comment:
How do you restrict access or did you mean restrict users from gaining root/administrative privileges i.e. "least privilege"

Reply:
In my view, this is both.
1. In a PaaS, usually, the access to underlying hosts is not shared with the user, and in control of the provider. 
2. Even in the provider side, the root access is something to be used if and only if needed, can be done by creating a sudo user. 
Does this help?